CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- Changed the order of `zero_grad` and `backward` calls in automatic optimization. Lightning now calls `zero_grad` before `backward` ([#6147](https://github.com/PyTorchLightning/pytorch-lightning/pull/6147))
+
```

Comment:
```suggestion
- Changed the order of `backward`, `step`, `zero_grad` to `zero_grad`, `backward`, `step` ([#6147](https://github.com/PyTorchLightning/pytorch-lightning/pull/6147))

```

Reply:
@awaelchli I applied the my above suggestion, how does it look now?

Let me note that the statement is fine for general users, but for some users who care what's really going on inside, the statement is actually false because `step` calls `zero_grad` and `backward` and updates params inside.

Reply:
Great. 

It just looks like that because of the closure, but the actual parameter update (step) comes at the end, so I don't see a problem.